app/src/main/java/com/github/houseorganizer/houseorganizer/panels/info/InfoActivity.java

```diff
@@ -62,6 +65,23 @@ protected void onCreate(Bundle savedInstanceState) {
         }
     }
 
+    public void showImage(View view){
+        if(currentHouse != null){
+            FirebaseStorage.getInstance()
+                    .getReference()
+                    .child("house_" + currentHouse.getId())
+                    .getDownloadUrl()
+                    .addOnCompleteListener(task -> {
+                if(task.isSuccessful()) {
+                    ImageHelper.showImagePopup(task.getResult(), view.getContext());
+                }
+                else {
+                    Toast.makeText(getApplicationContext(), "Could not find the attachment", Toast.LENGTH_SHORT).show();
```

Comment:
This might have to be put into a resource string so it can get translated, I just noticed its the same for attachments, we should look through our code to find these next week so its not that big a deal if its not done on this PR

Reply:
I agree!